src/contexts/SocketContext.tsx

```diff
@@ -66,29 +70,34 @@ export const SocketProvider: React.FC<TSocketProviderProps> = ({
     if (currentConfig.opfProvidedPredictions.length === 0) return
     const socketUrl =
       process.env.NEXT_PUBLIC_SOCKET_IO_URL || currentConfig.websocketURL
-    console.log('socketUrl', socketUrl)
+
     const newSocket = io(socketUrl, {
       path: '/api/datafeed',
-      transports: ['websocket']
+      transports: ['websocket'],
+      query: {}
     })
 
     setSocket(newSocket)
 
-    newSocket.on('newEpoch', (data: Maybe<TSocketFeedData>) => {
-      if (!data) return
+    newSocket.on(
+      `newEpoch-${timeFrameInterval}`,
```

Comment:
we should listed to events from all timeframes. For example when I'm on the 1h timeframe for more then 5m then go back to 5m websocket data is not updated and I have to refresh the page. It should be updated even if I'm on the 1h timeframe